rpi-convert-stage-5.sh

```diff
@@ -118,12 +123,21 @@ install_files() {
   sudo ln -sf /dev/null ${rootfs_dir}/etc/systemd/system/udisks2.service
 
   # Extract Linux kernel and install to /boot directory on rootfs
-  sudo cp ${boot_dir}/kernel7.img ${rootfs_dir}/boot/zImage
+  if [ "${device_type}" = "raspberrypi3" ]; then
```

Comment:
The same here, use a variable for kernel7.img / kernel.img file and call `sudo cp ...` only once. Having this you will also call a single command in line 135, instead of checking again `device_type`.